### DISCUSSION TITLE
GitHub Student Developer Pack - The status of my application is 'We're still processing your application' even after 30 days. Does this mean my application was rejected, or is there another reason for the delay?

### DISCUSSION THREAD

**Discussion by srinivask-1** [community/community#148000] [GitHub Education]
### Hi everyone,

I applied on November 26, 2024, and still haven't received any updates about the status of my application.

**Comment by davevad93** [comment]
Hi @srinivask-1 ,

This means that your application is still being reviewed, although it usually takes less time. When an application has been rejected, the following message appears:

<img width="296" alt="000000" src="https://github.com/user-attachments/assets/05c2b54b-2c76-4a0e-ada0-1dcbe69f70a7" />
<br>
<br>

When an application is approved, the status label is green and states: 

- `Congrats! Your request for @srinivask-1 was approved and your benefits will be available soon.`

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @srinivask-1 was approved and your benefits are now available.`

**Reply by srinivask-1** [reply]
What could be the reason for the delay? It would be great to know the reason for the delay. I hope it gets resolved soon.

**Reply by davevad93** [reply]
Hey @srinivask-1 , I can't say for sure, maybe it's because they have to review your documentation manually to check if it's valid, keep in mind that the support team can't speed up the review process, so opening a support ticket won't reduce the review time. However you have done the right thing by opening a discussion here.

**Reply by srinivask-1** [reply]
It is still showing the same status. Will it be approved?
